widgetry/src/widgets/spinner.rs

```diff
@@ -114,13 +148,21 @@ impl Spinner {
         spinner
     }
 
-    pub fn modify(&mut self, ctx: &EventCtx, delta: isize) {
+    pub fn modify(&mut self, ctx: &EventCtx, delta: T) {
         self.current += delta;
-        self.current = self.current.min(self.high);
-        self.current = self.current.max(self.low);
+        self.clamp();
         self.drawable = self.drawable(ctx.prerender, ctx.style());
     }
 
+    fn clamp(&mut self) {
+        if self.current > self.high {
```

Comment:
I was using `min` and `max`, but that's only there for `Ord`, and one of the callers wants to use `f64`, which is only `PartialOrd`